### PR TITLE
kibana support

### PR DESCRIPTION
**Pull request by vmitrevski**

### CODE REVIEW

File: tasks/ot-webpagetest.js [hunk 1]
```diff
@@ -28,12 +30,17 @@ module.exports = function(grunt) {
...
 
-                    if (options.notifyHipchat){
-                        notifyHipchat(message, options, done);
-                    }
-                    else{
-                        done();
-                    }
+                    async.series([
+			function(callback) {
+				 if (options.notifyHipchat) notifyHipchat(message, options, callback);
```
Comment (ryantomlinson) [inline tasks/ot-webpagetest.js]: Please use braces and new lines. It's neater and easier to read

File: tasks/ot-webpagetest.js [hunk 2]
```diff
@@ -28,12 +30,17 @@ module.exports = function(grunt) {
...
+				 else callback();
+			},
+			function(callback) {
+				if (options.notifyKibana) notifyKibana(data, options, callback);
```
Comment (ryantomlinson) [inline tasks/ot-webpagetest.js]: Same here

File: Gruntfile.js [hunk 3]
```diff
@@ -18,7 +18,10 @@ module.exports = function(grunt) {
                 wptApiKey: 'API_KEY_HERE',
                 hipchatApiKey: 'API_KEY_HERE',
                 roomId: 123456,
-                notifyHipchat: true
+                kibanaHost: 'localhost',
```
Comment (ryantomlinson) [inline Gruntfile.js]: Worth changing these parameters to "logstashHost" and "logstashPort" because it's really logstash support we're adding

**Comment by ryantomlinson** [comment]
@vmitrevski can you also update the README to say it now supports optional logstash logging. Thanks.

**Comment by bcatlin** [comment]
@ryantomlinson is this good to merge now?

**Comment by vmitrevski** [comment]
Hey Bryce, I’m having issues with getting it to log today so maybe we can hold off on the merge for a day or so?

Vasko Mitrevski
Engineer
vmitrevski@opentable.commailto:vmitrevski@opentable.com
[email_sig_R_154x40_v2]

**Comment by bcatlin** [comment]
OK, if you get it sorted out today please ping @ryantomlinson so we can get it merged.  Thanks.

**Comment by ryantomlinson** [comment]
@bcatlin @vmitrevski this doesn't pass jslint. Please correct outstanding issues.

**Comment by ryantomlinson** [comment]
@vmitrevski you have also mixed tabs and spaces. Please change your environment to use spaces.

**Pull request merged**